C/manual/ch_Finance-Quote.xml

```diff
@@ -177,30 +166,28 @@ SYNOPSIS
         </simpara>
       </step>
 
+      <step>
+        <simpara>Next, update &app-fq; with <userinput>sudo gnc-fq-update</userinput>.
+        </simpara>
+      </step>
+
       <step>
         <simpara>Run the <userinput>gnucash-cli --quotes info</userinput> command to verify that the program is
           already in a directory that is entered in the <envar>PATH</envar> environment variable.
           <footnote>
-            <simpara>If you&rsquo;ve installed &app; packages provided by your distribution,
-              <filename>gnucash-cli</filename> must be on your <envar>PATH</envar>. The currentness
-              of your distribution can be checked under
-              <ulink url="&url-repo;perl:finance-quote/versions"><citetitle>&app-fq;-
-              versions</citetitle></ulink>.
+            <simpara>If you&rsquo;ve installed &app; packages provided by your distribution, &app-cli; will be on your
```

Comment:
From [gnome's gdp-style-guide](https://developer-old.gnome.org/gdp-style-guide/2.32/locale-5.html.en)


Topic 14 | Contractions
-- | --
Rule | Replace contractions with the relevant complete words. 					 Especially avoid the 's contraction for 					 is and has. Translators can confuse 					 these contractions with each other, or with the Saxon genitive or even with 					 plurals. See also the entries for Apostrophe and 					 Contractions in Chapter 3 ― Grammar and Usage Guidelines.
Example | If you're new to GNOME...
Rewrite | If you are new to GNOME...
Exception | Only use a contraction if you cannot find a reasonable 					 alternative.

Perhaps a global change? 'grep -inr '&rsquo;' C/' returns several cases:
c/won\&rsquo;t/???
c/n\&rsquo;t/ not/
c/\&rsquo;ve/ have/
c/Let\&rsquo;s/Let us/
c/let\&rsquo;s/let us/

also wrong: `&rsquo;Parent Account&rsquo;`